src/components/manage/Controlpanels/Users/RenderUsers.jsx

```diff
@@ -101,9 +158,31 @@ class RenderUsers extends Component {
             </Dropdown.Menu>
           </Dropdown>
         </Table.Cell>
+        {Object.keys(this.state.user).length > 0 &&
+          this.props.userschema.loaded && (
+            <ModalForm
```

Comment:
I think the current `userschema` do respect the "Member fields" addition/deletion. So it's possible to populate with identical values both in user reg and user info. Perhaps those fields in the personal-information should be the same as user-info(clicking the edit button).